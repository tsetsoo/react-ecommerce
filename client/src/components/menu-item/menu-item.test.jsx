import { shallow } from 'enzyme'
import React from 'react'
import { MenuItem } from './menu-item.component'

it('expect to render MenuItem component', () => {
    const title = 'fake-item';
    expect(shallow(<MenuItem title={title} />)).toMatchSnapshot();
})