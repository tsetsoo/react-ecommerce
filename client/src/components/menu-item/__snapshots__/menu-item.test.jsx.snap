// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`expect to render MenuItem component 1`] = `ShallowWrapper {}`;
